docs/comparisons/math/quaternions.md

```diff
@@ -61,6 +61,27 @@ There was an [issue#25](https://github.com/JuliaSpace/ReferenceFrameRotations.jl
 {{badge Grassmann}}
 [Grassmann.jl](https://github.com/chakravala/Grassmann.jl) does not implement `Quaternion` as a struct, but quaternions are realized as an alias `Grassmann.Quaternion (alias for Spinor{V, T, 4} where {V, T})`.
 
+Comparison of quaternions in the context of `Grassmann` to other quaternion implementations is characterized by syntax design differences to fit into a larger mathematical formalism.
+Differences such as performance or edge cases can always be smoothed out, while syntax and design choices are fundamentally different.
+With `Grassmann` geometric algebra it is possible to achieve much more out of quaternions due to the immediate access to its fully general mathematical formalism.
+
+Quaternion algebra exists as a specialized sub-algebra within a more general `Grassmann` geometric algebra, where the goal is to seamlessly transition between quaternion abstractions and further combinatorial generalizations.
+There are several ways to assign `i,j,k` with `Grassmann` elements, perhaps the standard would be `i = v12`, `j = -v13`, `k = v23` (although this is not a unique choice).
+Since `j` and `v13` have opposite sign in this notation, the `quatvalues` method is exported to output the coefficients with the sign convention of `s,i,j,k`.
+* assign `Grassmann` elements for quaternion usage: `using Grassmann; basis"3"`
+* basis: `v, v₁, v₂, v₃, v₁₂, v₁₃, v₂₃, v₁₂₃` or `v, v1, v2, v3, v12, v13, v23, v123`
+* assign quaternion basis: `s, i, j, k = v, v12, -v13, v23` could be a standard choice
+* `quatvalues(::Quaternion)` returns the coefficient values according to `s, i, j, k`
+* `quaternion(s,i,j,k)` returns the `Grassmann` quaternion from the `s, i, j, k` standard
+
+As a result of the framework of geometric algebra implemented in `Grassmann`, vector algebra and quaternion algebra are compatible in a unified formalism.
+Given a quaternion operator `R` and a vector `x` the operator can be applied with either the `R>>>x` (evaluated as `R*x*conj(R)` operator) or `x⊘R` (evaluated as `conj(R)*x*R` operator) to transform vectors with quaternions.
+Converting a quaternion operator `R` on a three dimensional vector to a matrix (with the `x⊘R` evaluation)  can be done with `Matrix(operator(R))` for convenience.
+
+In the traditional quaternion packages there is only access to a very limited mathematical scope, which is similar to black and white images instead of full color.
+This developer (for example), will feel a loss of mathematical expressibility when downgrading from `Grassmann` to a more limited quaternion formalism, so the comparison is similar to choosing between black and white or full color image technology.
```

Comment:
I don't fully understand these sentences. Could you provide some concrete code examples to illustrate the "color" of Grassmann.jl? The analogy of black and white versus full color doesn't quite clarify the advantages for those unfamiliar with Grassmann.jl.

Reply:
Grassmann.jl has more colors because it is built on the full exterior algebra basis of `Grassmann` algebra, which encompasses far more than just quaternions in three dimensions. With Grassmann.jl you can generalize quaternions to any lower or higher dimensions, and not only does it support generalization of quaternions, it also supports reflections and not only rotations, it supports projective geometry and other kinds of geometrical transformations other than pure rotations that quaternions are enabling. So with `Grassmann` you get a full color vision, while with a pure quaternion package you are only limited to something like black and white colors.

Reply:
> The analogy of black and white versus full color doesn't quite clarify the advantages for those unfamiliar with Grassmann.jl.

Think of it like this: if you only know pure quaternions, then it is as if you can only see in black and white. You are unable to perceive that there are more colors that exist, so you cannot fathom what kind of an upgrade Grassmann.jl is without upgrading your cognition. After you learn lots of mathematics, you will eventually understand that pure quaternion packages are like black and white technology when compared with Grassmann.jl